### ISSUE TITLE
Feature Request: Parallel Multi-Drive Plotting

### ISSUE THREAD

**Issue by chadbaldwin** [Blagodarenko/miner-burst#24]
I know there hasn't been a new release in a few months, but just in case development is still going on, I thought I would recommend a feature that I think would be useful.

Parallel Multi-Drive Plotting.

While I'm plotting, one of the biggest bottlenecks I'm seeing is writing to my drives. I currently have two drives plotting at the same time right now, with equal resources 3 threads and 4GB RAM. While watching them plot, I notice that one drive will write very slowly for some reason, and then the plotter stops generating nonces while it waits for the scoops to finish writing. I feel like, while it's waiting for that drive to finish writing, that would be a good opportunity to crank up the threads for the other drive that is being plotted.

So basically, any time one plotting instance is stuck waiting for the scoops to finish writing, it will allow the other plotting instance to use those free threads.

**Comment by damccull** [comment]
Try putting this on his XPlotter repo instead of here. This is a miner.

**Comment by chadbaldwin** [comment]
Oh dammit, I wasn't paying attention and clicked on the wrong bookmark. Well that's annoying.
